cms/djangoapps/contentstore/views/course.py

```diff
@@ -102,8 +103,9 @@
 
 log = logging.getLogger(__name__)
 
-__all__ = ['course_info_handler', 'course_archive_handler', 'course_handler', 'course_listing',
-           'course_info_update_handler', 'course_search_index_handler',
+__all__ = ['course_info_handler', 'course_archive_handler', 'course_unarchive_handler',
```

Comment:
How about adding `'course_archive_handler', 'course_unarchive_handler'` in new line this way we will only have one new line in code diff instead of 2 deleted and 3 added lines of code.

Reply:
done